### ISSUE TITLE
Image refresh for rhel-7-4

### ISSUE THREAD

**Issue by cockpituous** [cockpit-project/cockpit#8253]
Image refresh for rhel-7-4
 * [ ] FAIL: image-refresh rhel-7-4

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-j7rg2.
Log: http://209.132.184.41/logs/image-refresh-8253-20171212-041717/

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-bb2fd.
Log: http://209.132.184.41/logs/image-refresh-8253-20171212-041717/

**Comment by cockpituous** [comment]
Task failed: http://209.132.184.41/logs/image-refresh-8253-20171212-041717/

**Comment by cockpituous** [comment]
image-refresh in progress on cockpit-tasks-cmm5w.
Log: http://fedorapeople.org/groups/cockpit/logs/image-refresh-8253-20171217-092626/

**Comment by cockpituous** [comment]
Task failed: http://fedorapeople.org/groups/cockpit/logs/image-refresh-8253-20171217-092626/

**Comment by martinpitt** [comment]
This is the error:
```
+ yum install -y epel-release
Error: Nothing to do
image-create: setup failed with code 1
[...]
Complete!
Loaded plugins: auto-update-debuginfo, product-id, search-disabled-repos,
              : subscription-manager
No package epel-release available.
```
It already failed a few days ago, so presumably not just a flake.

This seems to be my fault, I added/tested that on the centos-7 image only (forgetting that centos-7.setup is a symlink to rhel-7.setup). Eventually we want that COPR on the rhel-7 images as well, so let's just fix that properly. Later on in the setup file it already installs EPEL for mock, so this just needs to be moved around a little.

**Comment by martinpitt** [comment]
Fix in #8283.
